CONTRIBUTING.md

```diff
@@ -139,15 +142,15 @@ Commits should be signed. You can read more about [Commit Signing][] here.
 We recommend a read on our [Collaborator Guide](COLLABORATOR_GUIDE.md#accepting-modifications) for in-depth details on how we accept contributions into this repository. The list below describes some of the merging and approval rules adopted in this repository.
 
 - Pull Requests must be open for at least 48 hours (Or 72 hours if the PR was authored on the weekend).
-  - Pull requests might be immediately merged if they contain critical bug fixes, short erratas (e.g. typos from previous PRs) or any critical change that is considered a "showstopper" for the operation of the website.
-    - These kind of PRs should only be done by existing collaborators that have write-access and/or signed-off by administrators/maintainers.
+  - Pull requests might be immediately merged if they contain critical bug fixes, short errata (e.g. typos from previous PRs) or any critical change that is considered a "showstopper" for the operation of the website.
+    - This kind of PRs should only be done by existing collaborators that have write-access and/or signed off by administrators/maintainers.
```

Comment:
this change doesn't read well - suggest reverting or rewording if confusing. `kind of` is unnecessary

```
These pull requests
```

is unambiguous